ene-jun-2023/practicas/practica1/Practica 1 Andrea Garcia/practica1.py

```diff
@@ -0,0 +1,12 @@
+def maind():
+    inp = input('dame los numeros:')
+
+    espacio = inp.index(' ')
+    num1 = ''; num2 = ''
+
+    num1 = inp [:espacio]
+    num2 = inp [espacio:]
+
+    return "suma = ", (int(num1)+int(num2))
+
+print (maind())
```

Comment:
Si vas a tener una función principal y las vas a mandar a llamar desde el namespace global usa:
```python
if __name__ == "__main__":
  main()
```

Puedes ver por qué [aquí](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/)